index.html

```diff
@@ -208,22 +213,53 @@ <h1 class="menu-header">Buttons</h1>
             </div>
           </div>
           <div class="menu-item">
-            <h1 class="menu-header">Alignment</h1>
+            <h1 class="menu-header">Horizontal Alignment</h1>
+            <div class="radio-wrap">
+              <input id="control-layout-alignment-horizontal-left" class="control-layout-alignment-horizontal-left" type="radio" tabindex="1" name="control-layout-alignment-horizontal" value="left">
+              <label for="control-layout-alignment-horizontal-left"><span class="label-icon"></span>Left</label>
+            </div>
+            <div class="radio-wrap">
+              <input id="control-layout-alignment-horizontal-center" class="control-layout-alignment-horizontal-center" type="radio" tabindex="1" name="control-layout-alignment-horizontal" value="center">
+              <label for="control-layout-alignment-horizontal-center"><span class="label-icon"></span>Center</label>
+            </div>
+            <div class="radio-wrap">
+              <input id="control-layout-alignment-horizontal-right" class="control-layout-alignment-horizontal-right" type="radio" tabindex="1" name="control-layout-alignment-horizontal" value="right">
+              <label for="control-layout-alignment-horizontal-right"><span class="label-icon"></span>Right</label>
+            </div>
+          </div>
+          <div class="menu-item">
+            <h1 class="menu-header">Vertical Alignment</h1>
+            <p>Available when Bookmarks are note shown.</p>
```

Comment:
Typo in the word 'not'

Reply:
Good spot! I'll fix that.
This PR is not ready as I'm still working out some style fixes to the input range elements. 
Watch this space 👍